<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newAttributeWidget_2</class>
 <widget class="QDialog" name="newAttributeWidget_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Attribute</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Schema</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="schemaComboBox"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="tableNameLabel">
     <property name="text">
      <string>Table Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="tableComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="allTablesCheckBox">
     <property name="text">
      <string>Create attribute on all tables with geometries</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="AddAttributeWidget" name="addAttributeWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AddAttributeWidget</class>
   <extends>QWidget</extends>
   <header>DsgTools.gui.CustomWidgets.CustomDbManagementWidgets.addAttributeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
